tox.ini

```diff
@@ -45,3 +45,9 @@ setenv =
     PYTHONHASHSEED = 0
 commands =
     sphinx-build -n -T -b html -d {envtmpdir}/doctrees docs docs/_build/html
+
+[pytest]
+log_cli = 1
+log_cli_level = DEBUG
```

Comment:
```suggestion
# log_cli = true
# log_cli_level = DEBUG
```

I would rather comment this out for now. While it is helpful during debugging it is not quite ideal in all cases and can be done via command line interface also. Leaving it as a comment will maybe make it easier for people to find if they don't know pytest that well.

Could also remove it, up to you, but just don't want it using debug and doing live logging always.